frontend/src/app/(functional-pages)/teams/[id]/page.tsx

```diff
@@ -36,11 +36,11 @@ export default async function Page({ params }: any) {
       <Divider sx={{ borderBottomWidth: 3, my: 3 }} />
       <div className="grid grid-cols-2 gap-3">
         <InfoItem label="Contact Person" value= {contactPerson ?`${contactPerson?.firstName} ${contactPerson?.lastName}` : "N/A"} />
-        <InfoItem label="Leader" value= {leader[0]?`${leader[0].firstName} ${leader[0].lastName}`:"N/A"} />
-        <InfoItem label="Coach" value= {coach[0]?`${coach[0].firstName} ${coach[0].lastName}`:"N/A"} />
+        <InfoItem label="Leader" value= {leader.length>0? leader.map(l => `${l.firstName} ${l.lastName}`).join(", ") :"N/A"} />
+        <InfoItem label="Coach" value= {coach.length>0? coach.map(c => `${c.firstName} ${c.lastName}`).join(", ") :"N/A"} />
```

Comment:
Mangler lige at ændre dem her til leader og coach.

```suggestion
        <InfoItem label="Leader" value= {leader.length>0? leader.map(leader => `${leader.firstName} ${leader.lastName}`).join(", ") :"N/A"} />
        <InfoItem label="Coach" value= {coach.length>0? coach.map(coach => `${coach.firstName} ${coach.lastName}`).join(", ") :"N/A"} />
```

Reply:
.join(", ") er det for at sætte ", " bag på personen?